geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -5328,6 +5328,22 @@ public int getTotalNumberOfBuckets() {
     return this.totalNumberOfBuckets;
   }
 
+  /**
+   * This method returns a boolean to indicate if all server versions support Partition Region clear
+   */
+  public boolean allServerVersionsSupportPartitionRegionClear() {
+    for (int i = 0; i < getTotalNumberOfBuckets(); i++) {
```

Comment:
I am not very familiar with how PR is implemented, but is `getTotalNumberOfBuckets` returns the bucket counts for this server only or for the entire region?

Reply:
In my experience the whole region.